ocs_ci/deployment/deployment.py

```diff
@@ -1635,6 +1649,34 @@ def deploy_ocs(self):
                     update_ntp_compute_nodes()
                 assert ceph_health_check(namespace=self.namespace, tries=60, delay=10)
 
+        # In case of RDR, check for bluestore on osds: 4.14 onwards
+        if (
+            (version.get_semantic_ocs_version_from_config() >= version.VERSION_4_14)
+            and config.multicluster
+            and (config.MULTICLUSTER.get("multicluster_mode") == "regional-dr")
+        ):
+            if not ceph_cluster:
+                ceph_cluster = ocp.OCP(kind="CephCluster", namespace=self.namespace)
+            store_type = ceph_cluster.get().get("items")[0]["status"]["storage"]["osd"][
+                "storeType"
+            ]
+            if "bluestore-rdr" in store_type.keys():
+                logger.info("OSDs with bluestore found ")
+            else:
+                raise UnexpectedDeploymentConfiguration(
+                    f"OSDs were not brought up with bluestore! instead we have {store_type} "
```

Comment:
A minor change request:

```suggestion
                    f"OSDs were not brought up with Regional DR bluestore! instead we have {store_type} "
```